week06/6.1.count_specials.py

```diff
@@ -0,0 +1,32 @@
+#================================
+# 6.1 - ספירת מלאי
+#================================
+from check_type_decorator import check_type
+
+
+#--------------------------------------------------------------------------
+# return how many numbers that smaller than n and are divisible by 3 or 7
+#--------------------------------------------------------------------------
+@check_type(int)
+def count_specials(n):
+    s = set()
+    for i in range(1, n):
+        if i % 3 == 0 or i % 7 == 0:
```

Comment:
Try to avoid the usage of one char variable names

Reply:
Fixed now